index.html

```diff
@@ -4713,6 +4713,15 @@ <h2>Verification Method Revocation</h2>
 could discover a revoked key and use it to make cryptographically
 verifiable statements with a simulated date in the past.
 <br/><br/>
+For example, for a DID method that supports the `versionId` query parameter as
+well as the `updated` and `nextUpdate` DID document metadata properties, a
+verifier can validate a signature or proof of a revoked key iif: (1) the proof
+or signature contains the `versionId` to refer to the DID document that was
+used at the point the signature or proof was created. (2) The verifier can
+determine at what point in time the signature or proof was made (e.g. it was
+anchored on a blockchain). (3) For the DID document resolved using `versionId`,
+the `updated` timestamp is before 2 and the `nextUpdate` timestamp is after 2.
```

Comment:
I'm pretty sure `iif` was meant to mean "if and only if," in which case it should be `IFF`, or, better, fully written out.  Additionally, the three provisos either need careful re-punctuation to remain in a streaming paragraph, or to become the ordered list below.
```suggestion
For example, for a DID method that supports the `versionId` query parameter as
well as the `updated` and `nextUpdate` DID document metadata properties, a
verifier can validate a signature or proof of a revoked key IFF:
  <ol>
    <li>
The proof or signature contains the `versionId` to refer to the DID document
that was used at the point the signature or proof was created.
    </li>
    <li>
The verifier can determine at what point in time the signature or proof was
made, e.g., it was anchored on a blockchain.
    </li>
    <li>
For the DID document resolved using `versionId`, the `updated` timestamp is
before 2, and the `nextUpdate` timestamp is after 2.
    </li>
  </ol>
```

Reply:
Great suggestions, thanks!
Yep, was meant to be IFF :)